docs/source/notes/randomness.rst

```diff
@@ -54,18 +77,31 @@ When running on the cuDNN backend, two further options must be set::
     torch.backends.cudnn.deterministic = True
     torch.backends.cudnn.benchmark = False
 
+Alternatively, you can use the following, since cuDNN will be
+deterministic if either :code:`torch.set_deterministic(True)` or
+:code:`torch.backends.cudnn.deterministic = True` is set::
+
+    torch.set_deterministic(True)
+    torch.backends.cudnn.benchmark = False
+
 On some versions of cuDNN and CUDA, RNN and LSTM may have non-deterministic behavior.
 See :meth:`torch.nn.RNN` and :meth:`torch.nn.LSTM` for details and workarounds.
 
+When :code:`torch.backends.cudnn.benchmark = True` is set and cuDNN's convolution
```

Comment:
Going back to what we expect the user to know, would most users know they're using cuDNN?

Reply:
I've changed "cuDNN's convolution" to be just "CUDA's convolution". Do you think that's clear enough?